### ISSUE TITLE
What is 3.1415953?

### ISSUE THREAD

**Issue by TonyPeng-2018** [OpenDriveLab/UniAD#99]
From UniAD/projects/mmdet3d_plugin/models/utils/functional.py, I find a hardcoding value: 3.1415953. So I wanna know what it is and why you use this value.

def anchor_coordinate_transform(anchors, bbox_results, with_translation_transform=True, with_rotation_transform=True):
    """
    Transform anchor coordinates with respect to detected bounding boxes in the batch.

    Args:
        anchors (torch.Tensor): A tensor containing the k-means anchor values.
        bbox_results (List[Tuple[torch.Tensor]]): A list of tuples containing the bounding box results for each image in the batch.
        with_translate (bool, optional): Whether to perform translation transformation. Defaults to True.
        with_rot (bool, optional): Whether to perform rotation transformation. Defaults to True.

    Returns:
        torch.Tensor: A tensor containing the transformed anchor coordinates.
    """
    batch_size = len(bbox_results)
    batched_anchors = []
    transformed_anchors = anchors[None, ...] 
    for i in range(batch_size):
        bboxes, scores, labels, bbox_index, mask = bbox_results[i]
        yaw = bboxes.yaw.to(transformed_anchors.device) 
        bbox_centers = bboxes.gravity_center.to(transformed_anchors.device) 
        if with_rotation_transform: 
            **_angle = yaw - 3.1415953_**
            rot_yaw = rot_2d(angle) 
            rot_yaw = rot_yaw[:, None, None,:, :]
            transformed_anchors = rearrange(transformed_anchors, 'b g m t c -> b g m c t')  
            transformed_anchors = torch.matmul(rot_yaw, transformed_anchors)
            transformed_anchors = rearrange(transformed_anchors, 'b g m c t -> b g m t c')
        if with_translation_transform:
            transformed_anchors = bbox_centers[:, None, None, None, :2] + transformed_anchors
        batched_anchors.append(transformed_anchors)
    return torch.stack(batched_anchors)

**Comment by Yihanhu** [comment]
Hi @TonyPeng-2018,

This is to align the bounding box coordinate system with the ego coordinate system.

**Comment by TonyPeng-2018** [comment]
@Yihanhu Thanks!
